perun-cli/Perun/beans/Attribute.pm

```diff
@@ -332,13 +342,37 @@ sub setValueFromArray {
 	}
 }
 
+sub isUnique
+{
+	my $self = shift;
+
+	return ($self->{_unique}) ? 'true' : 'false';
```

Comment:
We should return "0" instead false and "1" instead true. Both Strings "true" and "false" are in perl evaluate as boolean "true". Also you can't just evaluate $self->{_unique} because if there is object 'JSON::XS::false' it will be evaluate as boolean 'true'.

Reply:
The JSON::XS::false is really false but only if you have "use JSON::XS" in libs. So you can skip this problem, it is evaluated correctly there.